### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by customcommander**
Couple of quick wins (I hope) to help clear a few documentation-related issues. See issues referenced in the commit messages for context.

### CODE REVIEW

File: source/composeWith.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import reverse from './reverse.js';
 
 /**
  * Performs right-to-left function composition using transforming function. The last function may have
- * any arity; the remaining functions must be unary.
+ * any arity; the remaining functions must be unary. Unlike `compose` functions are passed in an array.
```
Comment (adispring) [inline source/composeWith.js:8]: Unlike `compose`, functions are passed in an array.

Add a comma after `compose`, may be more clear?

**Review by adispring — CHANGES_REQUESTED** [review]

File: source/composeWith.js (hunk 1 above)
Comment (customcommander) [reply source/composeWith.js]: Done.

**Review by adispring — DISMISSED** [review]

File: source/memoizeWith.js [hunk 2]
```diff
@@ -4,39 +4,47 @@ import _has from './internal/_has.js';
...
+ * Care must be taken when implementing `sig` to avoid key collision,
  * or if tracking references, memory leaks and mutating arguments.
  *
  * @func
  * @memberOf R
  * @since v0.24.0
  * @category Function
  * @sig (*... -> String) -> (*... -> a) -> (*... -> a)
- * @param {Function} fn The function to generate the cache key.
+ * @param {Function} sig The function to generate the cache key.
```
Comment (CrossEye) [inline source/memoizeWith.js:26]: Would `kegGen` be a more explicit name?

Comment (customcommander) [reply source/memoizeWith.js:26]: You mean `keyGen`? Sure that sounds good to me. Will change.

Comment (customcommander) [reply source/memoizeWith.js]: Done.

**Review by CrossEye — APPROVED** [review]

**Pull request merged**